.github/workflows/deploy-docs.yml

```
@@ -11,12 +11,12 @@ jobs:
 
     steps:
     - name: checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3.1.0
```

We can use all "3.*" instead of a point release.

```suggestion
      uses: actions/checkout@v3
```